chapters/annotations.tex

```diff
@@ -106,13 +106,12 @@ \subsection{Annotations for Figures}\label{annotations-for-figures}
 described in \cref{variable-replacements}, can be used in the
 \lstinline!title! of \lstinline!Figure! and \lstinline!Plot!.
 
-The \lstinline!identifier! in \lstinline!Figure! and \lstinline!Plot! is an optional \lstinline!String! identifier, and is intended to identify the
-\lstinline!Figure! and \lstinline!Plot! for programmatic access.  The \lstinline!figures! annotation is inherited in the sense that each class has
-a collection of figures comprised by the contents of the \lstinline!figures! annotation in the class itself, as well as the \lstinline!figures! annotations
-from any base classes.  A \lstinline!Figure! must be uniquely identified by its \lstinline!identifier! and a class having it in its collection.  This
-means that a \lstinline!Figure! \lstinline!identifier! must be unique among all \lstinline!Figure! annotations within the same \lstinline!figures!
-annotation as well as among all \lstinline!figures! annotations from inherited classes.  A \lstinline!Plot! \lstinline!identifier! on the other hand
-is only required to be unique among the \lstinline!plots! in the the same \lstinline!Figure! annotation.
+The \lstinline!identifier! in \lstinline!Figure! and \lstinline!Plot! is a \lstinline!String! identifier, and is intended to identify the \lstinline!Figure! and \lstinline!Plot! for programmatic access.
+The \lstinline!figures! annotation is inherited in the sense that each class has a collection of figures comprised by the contents of the \lstinline!figures! annotation in the class itself, as well as the \lstinline!figures! annotations from any base classes.
+A \lstinline!Figure! must be uniquely identified by its \lstinline!identifier! and a class having it in its collection.
+This means that a \lstinline!Figure! \lstinline!identifier! must be unique among all \lstinline!Figure! annotations within the same \lstinline!figures! annotation as well as among all \lstinline!figures! annotations from inherited classes.
+A \lstinline!Plot! \lstinline!identifier! on the other hand is only required to be unique among the \lstinline!plots! in the the same \lstinline!Figure! annotation.
+If the identifier is an empty string it cannot be used for programmatic access and is exempt from the uniqueness requirements.
```

Comment:
Since all other occurrences of _identifier_ are inline code in this paragraph, I suppose the last one should be too.  We could also say _an_ instead of _the_ to reduce risk of interpreting this as only applying to `Plot`:
```suggestion
If an \lstinline!identifier! is an empty string it cannot be used for programmatic access and is exempt from the uniqueness requirements.
```